cluster/gce/util.sh

```diff
@@ -898,7 +907,21 @@ REGION: $(yaml-quote ${REGION})
 VOLUME_PLUGIN_DIR: $(yaml-quote ${VOLUME_PLUGIN_DIR})
 KUBELET_ARGS: $(yaml-quote ${KUBELET_ARGS})
 REQUIRE_METADATA_KUBELET_CONFIG_FILE: $(yaml-quote true)
+ENABLE_NETD: $(yaml-quote ${ENABLE_NETD:-false})
+CUSTOM_NETD_YAML: |
+$(echo "${CUSTOM_NETD_YAML:-}" | sed -e "s/'/''/g")
 EOF
+  if [[ ${ENABLE_NETD:-} == "true" &&  "${master}" == "false" ]]; then
```

Comment:
why we need this? I believe we already handle the network plugin settings in construct-kubelet-flags()

Reply:
We could set NON_MASQUERADE_CIDR independently from netd. This is how we did it in GKE API server.